fluent/src/builtins.js

```diff
@@ -27,22 +27,18 @@ function values(opts) {
 
 export
 function NUMBER([arg], opts) {
-  if (arg instanceof FluentNone) {
-    return arg;
-  }
   if (arg instanceof FluentNumber) {
     return new FluentNumber(arg.valueOf(), merge(arg.opts, opts));
   }
-  return new FluentNone("NUMBER()");
```

Comment:
This was introduced in April in https://github.com/projectfluent/fluent.js/commit/24355cf9aa01ef19b72613c0d7423c978bf51490 to provide better fallback values. Today we can simply throw :)